paddle/fluid/framework/ir/mkldnn/cpu_bfloat16_pass.cc

```diff
@@ -199,11 +199,7 @@ class DeQuantizer final : public Quanter {
   bool IsNotPermittedName(const std::string& output_name) const override {
     std::unordered_map<std::string, std::vector<std::string>> block_list{
         {"layer_norm",
-         {"Mean", "Variance"}},        // not used in inference in MKLDNN
-        {"fc", {"ResidualData"}},      // artifical output, already dequantized
-        {"matmul", {"ResidualData"}},  // artifical output, already dequantized
-        {"matmul_v2",
-         {"ResidualData"}}};  // artifical output, already dequantized
+         {"Mean", "Variance"}}};  // not used in inference in MKLDNN
```

Comment:
Could you update this comment? (I won't force it)
```suggestion
         {"Mean", "Variance"}}};  // not used in inference in oneDNN
```